src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/ReadyToRunLibraryRootProvider.cs

```diff
@@ -14,11 +17,13 @@ public class ReadyToRunRootProvider : ICompilationRootProvider
     {
         private EcmaModule _module;
         private ProfileData _profileData;
+        private TargetDetails _targetDetails;
 
-        public ReadyToRunRootProvider(EcmaModule module, ProfileDataManager profileDataManager)
+        public ReadyToRunRootProvider(EcmaModule module, ProfileDataManager profileDataManager, TargetDetails targetDetails)
```

Comment:
Nit - you don't need the extra targetDetails parameter, you can get to it through module.Context.Target.